docs/content/v0.20/packaging.md

```diff
@@ -0,0 +1,28 @@
+# Packaging and Delivery
+
+## status-addon
+
+The [status-addon](https://github.ibm.com/dettori/status-addon) repo is the source of a RedHat-style operator. 
+
+The operator is delivered by a Helm chart at `quay.io/pdettori/status-addon-chart`. Chart versioning? How is the chart built, pushed?
+
+There are probably one or more container images involved.
+
+## KubeStellar
+
+### Central container image
+
+Appears to be at `ghcr.io/kubestellar/kubestellar/kubestellar-operator:0.20.0-alpha.1`. `make ko-build-push` will create that image. `make ko-build-local` will make a local image for just the local platform.
+
+### KubeStellar Operator Helm Chart
+
+The source for a Helm chart is in [core-helm-chart](../../../core-helm-chart). `make chart` (re)derives it from loal sources.
+
+This chart creates (among other things) a `Deployment` object that runs a container from the image `ghcr.io/kubestellar/kubestellar/kubestellar-operator:0.20.0-alpha.1`.
+
+### KubeFlex PostCreateHooks
+
+There are two `PostCreateHook` objects defined in [config/postcreate-hooks](../../../config/postcreate-hooks).
+
+- `ocm.yaml` adds `clusteradm` by running a container using the image `quay.io/kubestellar/clusteradm:0.7.1` which is built from ... what?
+- `kubestellar.yaml` runs container image `quay.io/kubestellar/helm:v3.13.2` (which is built by what? From what?) to instantiate the chart from `oci://ghcr.io/kubestellar/kubestellar/kubestellar-operator-chart` (tag?), which is built by (what?) from (what? probably answered above).
```

Comment:
Similar answer to above. No community image is available for helm so we build our own from https://github.com/helm/helm using ko. We should also ad a script for that - perhaps @francostellari can create one based on the one he ddid for clusteradm.

Reply:
thanks, noted.

Reply:
@pdettori the current version that you have is also single arch. I can create script and build the multi arch. Do you specifically want v3.13.2 or latest is good?

Reply:
Please see https://github.com/kubestellar/kubestellar/pull/1604/files